src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -2366,8 +2366,22 @@ public unsafe IEnumerable<int> M4(int* a)
             })();
     }
 }";
-            CreateCompilation(src, options: TestOptions.UnsafeDebugDll)
-                .VerifyDiagnostics(
+            var comp = CreateCompilation(src, options: TestOptions.UnsafeDebugDll, parseOptions: TestOptions.RegularNext.WithFeature("run-nullable-analysis", "never"));
```

Comment:
Why skip nullable analysis?

Reply:
> Why skip nullable analysis?

Because it binds the body before we can get the symbol, which defeats the purpose of the test.